packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Customize the Okta-hosted error pages
+title: Customize the error pages
```

Comment:
Maybe simplify to "Customize error pages"?

Reply:
we need to doc they should not be using `meta` tags to customize the csp on customizable pages and we also need to test the implications of this cc @kevinroh-okta @zeusandrade-okta @bhagyaprabhakar-okta @bryanhonan-okta @rishabhchandna-okta

Reply:
https://okta.slack.com/archives/CUK1VSYTZ/p1675372178689119?thread_ts=1675368555.780589&cid=CUK1VSYTZ

Reply:
@mihaiiacob-okta Thank you! I have another jira for it that I'm adding to this PR.